src/cloud-api-adaptor/test/provisioner/provision.go

```diff
@@ -559,27 +559,6 @@ func (p *CloudAPIAdaptor) Deploy(ctx context.Context, cfg *envconf.Config, props
 		return err
 	}
 
-	log.Info("Installing peerpod-ctrl")
-	cmd = exec.Command("make", "-C", "../peerpod-ctrl", "deploy")
-	// Run the deployment from the root src dir
-	cmd.Dir = p.rootSrcDir
-	// Set the KUBECONFIG env var
-	cmd.Env = append(os.Environ(), fmt.Sprintf("KUBECONFIG="+cfg.KubeconfigFile()))
-	stdoutStderr, err = cmd.CombinedOutput()
-	log.Tracef("%v, output: %s", cmd, stdoutStderr)
-	if err != nil {
-		return err
-	}
-
-	// Wait for the peerpod-ctrl deployment to be ready
```

Comment:
Shouldn't it still wait the peerpod-ctrl deployment be ready? Sorry, never tried that new install method so I don't know if any other previous condition already ensure peerpod-ctrl is up and running at this point.

Reply:
That's a fair question. My understanding (which could be wrong) is that as the peerpod-ctrl is deployed in the operator stage, not at the end, we would have expected it to be running by this point, but I don't think there is any harm to wait in case it hasn't, so I'll try and re-add that.

Reply:
Thanks for raising this - after looking into it deeper I've found that the operator isn't creating the peerpod-ctrl properly with the error:
```
2024-05-14T08:12:27Z	ERROR	Reconciler error	{"controller": "peerpod", "controllerGroup": "confidentialcontainers.org", "controllerKind": "PeerPod", "PeerPod": {"name":"nginx-resource-xlctp","namespace":"default"}, "namespace": "default", "name": "nginx-resource-xlctp", "reconcileID": "399c7b13-cc9d-4aba-ade8-d332ac01d62a", "error": "libvirt cloud provider not supported"}
```
so that's something else that needs debugging

Reply:
I will push the re-addition of this code with a WIP label as it shows it's important to test, but I think the deployment name we are looking for will be likely change.

Reply:
I'm not sure if there is anyway we can reasonably test for this with the e2e tests though?